docs/rest_api.md

```diff
@@ -416,6 +416,12 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
```

Comment:
All of the other driver-specific stuff is documented separately, c.f. channel formats. We ought to do the same with obm-specific things like available boot devices. And yes, each one needs to be described.